components/fxa-client/ffi/src/lib.rs

```diff
@@ -237,6 +237,7 @@ pub extern "C" fn fxa_migrate_from_session_token(
     session_token: FfiStr<'_>,
     k_sync: FfiStr<'_>,
     k_xcs: FfiStr<'_>,
+    in_place_migration: bool,
```

Comment:
@rfk any ideas what we can call this?

Reply:
Hrm. I'm in the camp that says boolean flag parameters like this are often a public API anti-pattern. I also think the current API is using the word "migration" to mean something closer to "import" or "copy".  (In other words, someone reading the name "migrateFromSessionToken" could reasonable conclude that it is in-place by default).

Do we have any consumers that are going to call this with `in_place_migration=false` in practice?

If so, would you consider exposing two functions, `migrateFromSessionToken` for doing an in-place migration and something like `importFromSessionToken` for doing the copying version?

Reply:
> Do we have any consumers that are going to call this with in_place_migration=false in practice?

Lockwise might, maybe 'migrate' is not the right naming here and heavily attached to Fennec->Fenix migration